packages/docusaurus/src/commands/start.ts

```diff
@@ -117,6 +117,12 @@ export async function start(
   );
 
   let config: webpack.Configuration = merge(await createClientConfig(props), {
+    watchOptions: {
+      ignored: [siteDir+'/**/node_modules'],
+      poll: Number.isInteger(cliOptions.poll)
+        ? cliOptions.poll
+        : false,
```

Comment:
```suggestion
      poll: cliOptions.poll,
```

poll accepts boolean | number | undefined which match our option type